specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-03-01-preview/mongoCluster.json

```diff
@@ -0,0 +1,983 @@
+          "pattern": "^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$"
+        },
+        "endIpAddress": {
+          "type": "string",
+          "description": "The end IP address of the mongo cluster firewall rule. Must be IPv4 format.",
+          "pattern": "^(([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])\\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5])$"
+        }
+      },
+      "required": [
+        "startIpAddress",
+        "endIpAddress"
+      ]
+    },
+    "FirewallRule": {
+      "type": "object",
+      "description": "Represents a mongo cluster firewall rule.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/FirewallRuleProperties",
+          "x-ms-client-flatten": true,
+          "description": "The properties of a firewall rule."
+        }
+      },
+      "required": [
+        "properties"
+      ]
+    },
+    "FirewallRuleListResult": {
+      "type": "object",
+      "description": "A list of firewall rules.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The list of firewall rules in a mongo cluster.",
+          "items": {
+            "$ref": "#/definitions/FirewallRule"
+          }
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ConnectionString": {
+      "type": "object",
+      "description": "Connection string for the mongo cluster",
+      "properties": {
+        "connectionString": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Value of the connection string"
+        },
+        "description": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Description of the connection string"
+        }
+      }
+    },
+    "ListConnectionStringsResult": {
+      "type": "object",
+      "description": "The connection strings for the given mongo cluster.",
+      "properties": {
+        "connectionStrings": {
+          "readOnly": true,
+          "type": "array",
+          "description": "An array that contains the connection strings for a mongo cluster.",
+          "x-ms-identifiers": [],
+          "items": {
+            "$ref": "#/definitions/ConnectionString"
+          }
+        }
+      }
+    },
+    "MongoClusterRestoreParameters": {
+      "type": "object",
+      "description": "Parameters used for restore operations",
+      "properties": {
+        "pointInTimeUTC": {
+          "type": "string",
+          "format": "date-time",
+          "description": "UTC point in time to restore a mongo cluster"
```

Comment:
>mongo cluster" [](http://example.com/codeflow?start=57&length=14)

So we partnered with Mongo on this is that right? In docs, brandingwise, are we happy with calling them 'mongo clusters' in the everyday conversational fashion, or would Mongo want something more branding-conscious like 'MongoDB clusters' or 'Mongo clusters?'

Reply:
(any promises to them we need to keep?)